Makefile

```diff
@@ -495,4 +496,26 @@ validate-release: setup-venv
 ## prepare files for OperatorHub PR
 ## use this target when the operator needs to be released as upstream operator
 prepare-operatorhub-pr:
-	./hack/prepare-operatorhub-pr.sh $(OPERATOR_VERSION) $(OPERATOR_BUNDLE_IMAGE_REF)
\ No newline at end of file
+	./hack/prepare-operatorhub-pr.sh $(OPERATOR_VERSION) $(OPERATOR_BUNDLE_IMAGE_REF)
+
+.PHONY: prepare-release-manifests
+## prepare file for release
+## use this target when the operator needs to be released as a single manifest with project release tags
+prepare-release-manifests: setup-venv
+	./hack/prepare-operatorhub-pr.sh $(OPERATOR_VERSION) $(OPERATOR_BUNDLE_IMAGE_REF)
```

Comment:
we have already target for this and we should declare it as dependency to `prepare-release-manifests`